projects/igniteui-angular/src/lib/grids/toolbar/grid-toolbar.base.ts

```diff
@@ -79,3 +130,16 @@ function _makeOverlaySettings(): OverlaySettings {
         closeOnOutsideClick: true
     };
 }
+
+export abstract class BaseToolbarColumnActions extends BaseToolbarDirective {
+
+    protected columnActionsUI: IgxColumnActionsComponent;
```

Comment:
This should be public so that users can change all the column action components properties by a ref to its toolbar wrapper.

Reply:
That was the first intention, not exposing those references directly to the users but just the API they would need.
That was the approach we've discussed with @rkaraivanov, building a facade around the toggle and column actions functionalities.

Reply:
Sure. But then you have to expose more stuff. The customer wants to reset the `filterCriteria` on closing so you need to expose that property from the actions component at least. I'd probably expose all public properties that control its state just to be on the safe side.

Reply:
Yep, I understand. That is why the PR is in a DRAFT state as it is not ready and it just serves as a starting point of discussion. I will definitely expose some more methods.